Source/Core/Common/CDUtils.cpp

```diff
@@ -208,7 +208,7 @@ std::vector<std::string> cdio_get_devices()
 // Returns true if device is a cdrom/dvd drive
 bool cdio_is_cdrom(std::string device)
 {
-#ifndef _WIN32
+#ifdef _WIN32
```

Comment:
The code contained in that ifndef is meant to be included on platforms that aren't windows... I imagine this would include freebsd?